src/ts/state/reducer.ts

```diff
@@ -122,8 +122,10 @@ export interface IPluginState<TMatches extends IMatch = IMatch> {
   requestsInFlight: {
     [requestId: string]: IBlocksInFlightState;
   };
-  // The current error status.
-  error: string | undefined;
+  // The current error message.
+  errorMessage: string | undefined;
+  // The current error state.
+  hasError: boolean;
```

Comment:
Interesting ... is it possible to be in an error state without a message (`errorMessage: undefined`)? If not, is `hasError` redundant? (This may be inconsequential re: other changes we're about to make, so it's only a nit!)

Reply:
I have replaced this with a selector per our conversation earlier.